roofit/roofitcore/inc/RooAbsReal.h

```diff
@@ -380,10 +380,16 @@ class RooAbsReal : public RooAbsArg {
   TString integralNameSuffix(const RooArgSet& iset, const RooArgSet* nset=nullptr, const char* rangeName=nullptr, bool omitEmpty=false) const ;
 
 
-  bool isSelectedComp() const ;
+
 
 
  public:
+  bool isSelectedComp() const ;
+  void selectComp(bool flag) {
+     // If flag is true, only selected component will be included in evaluates of RooAddPdf components
+     _selectComp = flag ;
+  }
```

Comment:
```suggestion
  /// If flag is true, only selected component will be included in evaluates of RooAddPdf components
  void selectComp(bool flag) {
     _selectComp = flag ;
  }
```
Don't bother doing it now, but for the next time if you are changing the code and stumble across the old documentation strings, please take the opportunity to update them to the doxygen format (`///` and `.` at the end).